[Support] Tenant & Principal Aware Cache Invalidation (#100)
SAP/cloud-sdk: docs/java/features/resilience/caching.mdx

                                                     .registerCacheEntryListener(SimpleCacheEntryListener); //Registers the event listener
```

> where `SimpleCacheEntryListener` is a class that implements the interfaces of the required events w.r.t cache defined in [EventType](https://javadoc.io/doc/javax.cache/cache-api/latest/javax/cache/event/EventType.html).

Reviewer: ```suggestion
where `SimpleCacheEntryListener` is a class that implements the interfaces of the required events with respect to cache defined in [EventType](https://javadoc.io/doc/javax.cache/cache-api/latest/javax/cache/event/EventType.html).
```
